### DISCUSSION TITLE
Map - Map APIs

### DISCUSSION THREAD

**Discussion by KarlHanEdn** [CMPUT301W22T35/QRcode-Quest#55] [General]
Google Maps or OpenStreetMap, which map api should we use?
(Assuming we are using Google Maps) Which particular api of Google Maps should be chosen? We probably should choose one based on the cost (implementation difficulty and, of course, billing) and the looks of the map.

**Comment by KarlHanEdn** [comment]
We talked about this in today's meeting, and Google Maps seems like the more straight forward way to go. The api is not free but there is a free amount that is allowed per month.

**Comment by RottenFishbone** [comment]
I am _much_ more partial to using google maps, out google's support for it. There is a MapView object provided by their maps SDK.

The API for that is "Maps SDK for Android"
